docs/react-v9/contributing/new-release-process-v9-packages.md

```diff
@@ -160,8 +162,10 @@ yarn nx g @fluentui/workspace-plugin:prepare-initial-release --project @fluentui
 
 > automation status: implemented ✅ / https://github.com/microsoft/fluentui/pull/28505
 
+You can use `yarn generate` or Nx Console to navigate through the process.
+
 ```sh
-yarn nx g @fluentui/workspace-plugin:prepare-initial-release --project @fluentui/react-one-preview --phase=stable
+yarn nx g @fluentui/workspace-plugin:prepare-initial-release --project react-one-preview --phase=stable
```

Comment:
we need to update also previous commands in kickof an preview phase chapter.

I'm fine having the mantion about yarn generate and nx console only once before the kickoff phase , something like 

```md
> [!TIP]
> you can use `yarn generate` or Nx Console to navigate through the process.

```

Reply:
fixed